### PR TITLE
Update Kruize image with latest commit

### PR DESCRIPTION
**Pull request by khansaad**
This PR updates the Kruize image with the latest commit ID.

### CODE REVIEW

File: kruize-clowdapp.yaml [hunk 1]
```diff
@@ -157,7 +157,7 @@ parameters:
 - description: Kruize image tag
   name: KRUIZE_IMAGE_TAG
   required: true
-  value: "870ea25"
+  value: "69020a42"
```
Comment (kgaikwad) [inline kruize-clowdapp.yaml:160]: @khansaad  Thank you for pull-request. Observation - I don't see any image created with this refs number under [cloudservices/autotune](https://quay.io/repository/cloudservices/autotune?tab=tags). Also, it includes just first 7 digits.

Comment (khansaad) [reply kruize-clowdapp.yaml]: Updated the digits count to first 7.

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:

**Comment by kgaikwad** [comment]
@dgaikwad  could you please check and confirm reason behind test pipeline failure? can we ignore test failure? It seems irrelevant to PR change.

```
application = <iqe.base.application.Application object at 0x7f41b0ae2310>
11:19:59 source_and_upload_data_with_n_days = {'boto3_session_rosocp': (s3.ServiceResource(), <botocore.client.S3 object at 0x7f41aebf91f0>, s3.Bucket(name='ros-eph...314_openshift_report.4.csv?AWSAccessKeyId=sse9T4LPLFA6&Signature=YYan70YFHlWtfUfLk%2F9ziDmmKuE%3D&Expires=1698835762']}
11:19:59 
11:19:59     @pytest.mark.parametrize("n_days_data", [1, 7, 15, 30], indirect=True)
11:19:59     def test_get_recommendation_list(application, source_and_upload_data_with_n_days):
11:19:59         """Test that will test list of ocp recommendations
11:19:59 
11:19:59         GET /api/cost-management/v1/recommendations/openshift/
11:19:59 
11:19:59         metadata:
11:19:59             assignee: dgaikwad
11:19:59             importance: high
11:19:59 
11:19:59         """
11:19:59         if application.config.current_env in ["stage_proxy", "stage", "prod"]:
11:19:59             source_and_upload_data_with_n_days.list_recommendations_by_cluster()
11:19:59         payload = {"_return_http_data_only": False}
11:19:59         response = application.ros_ocp.rest_client.default_api.get_recommendation_list(**payload)
11:19:59         assert response[1] == 200
11:19:59 >       assert response[0].data
11:19:59 E       AssertionError: assert []
11:19:59 E        +  where [] = {'data': [],\n 'links': {'first': '/api/cost-management/v1/recommendations/openshift?limit=10&offset=0',\n           'last': '/api/cost-management/v1/recommendations/openshift?limit=10&offset=10'},\n 'meta': {'count': 0, 'limit': 10, 'offset': 0}}.data
11:19:59 
11:19:59 /iqe_venv/lib/python3.9/site-packages/iqe_ros_ocp/tests/rest/test_endpoints.py:28: AssertionError
```

**Pull request merged**